index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <nav id="navcontainer">
+    <div>
+      <h3 id="menulogo">John Doe</h3>
+    </div>
+    <div>
+      <img id="burgermenu" src="./img/burger.png" alt="">
+    </div>
+  </nav>
+
+  <header>
+    <div id="headercontainer">
+      <h1 id="hero">Hey there. I'm Mauricio</h1>
+      <h3 id="subhero">I am a Software Developer</h3>
+      <p id="herotext">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.</p>
+      <div>
+        <img src="./img/social/github.png" alt="">
+        <img src="./img/social/linkedin.png" alt="">
+        <img src="./img/social/scissors.png" alt="">
+        <img src="./img/social/twitter.png" alt="">
+        <img src="./img/social/m.png" alt="">
+      </div>
```

Comment:
- [ ] Great job so far, however, here since we're presenting a  list of icons it's more sematic to use a list tag.
- [ ] It's one of the project requirements to make the icons a link, kindly fix that.
```
<ul>
 <li> <a>...</a></li>
 <li> <a>...</a></li>
</ul>
```